### PR TITLE
tny.so

### LINKED ISSUE #78325: tny.so
### Issue URL (Anti Adblock Script)
 [https://tny.so/3AKErC9b?](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftny.so%2F3AKErC9b%3F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>Fanboy's Enhanced Tracking List, <br/>Fanboy's Anti-Facebook List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies<br/><br/><b>Security:</b><br/> Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Spanish/Portuguese, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>Iranian, <br/>Polish Anti Adblock Filters

### PR DESCRIPTION
**Pull request by wout-codes**
#78325

### CODE REVIEW

**Comment by wout-codes** [comment]
WIP

**Comment by wout-codes** [comment]
@zloyden Done.

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
...
 !
 delfi.ee##.C-article-social
+down.mdiaload.com,down.fast-down.com##tbody > tr:nth-of-type(5)
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:42]: Better `down.mdiaload.com,down.fast-down.com#?#tbody > tr div.share:upward(2)`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:42]: I don't prefer Ext. CSS in this case. Simple rules are enough, in my perspective.

Comment (zloyden) [reply SocialFilter/sections/specific.txt:42]: But not with this modifier.

Comment (wout-codes) [reply SocialFilter/sections/specific.txt:42]: I don't get it, is this modifier Ext. CSS or normal modifiers?

Comment (zloyden) [reply SocialFilter/sections/specific.txt]: `ext-css` is more accurate here.

**Review by zloyden — APPROVED** [review]

**Pull request merged**